store/tikv/client_batch.go

```diff
@@ -215,8 +215,8 @@ func (c *batchCommandsClient) send(request *tikvpb.BatchCommandsRequest, entries
 		c.batched.Store(requestID, entries[i])
 	}
 	if err := c.client.Send(request); err != nil {
-		logutil.BgLogger().Error(
-			"batch commands send error",
+		logutil.BgLogger().Warn(
```

Comment:
can we set it to `debug` or `info`? or can we not print this message since the user no need to worry about these kinds of logs?

Reply:
I prefer to set it to `info` if you think it's better.